backend/coreapp/serializers.py

```diff
@@ -50,19 +50,9 @@ class ScratchCreateSerializer(serializers.Serializer[None]):
     diff_label = serializers.CharField(allow_blank=True, required=False)
 
 class ScratchSerializer(serializers.ModelSerializer[Scratch]):
-    class Meta:
-        model = Scratch
-        fields = ["slug", "name", "description", "compiler", "platform", "compiler_flags", "target_assembly", "source_code", "context", "diff_label", "score", "max_score"]
-
-# XXX: ideally we would just use ScratchSerializer, but adding owner and parent breaks creation
-class ScratchWithMetadataSerializer(serializers.ModelSerializer[Scratch]):
     owner = ProfileField(read_only=True)
-    parent = serializers.HyperlinkedRelatedField( # type: ignore
```

Comment:
What is the shape of `parent` now this is removed?

Reply:
I believe it's just the primary key atm, but we can increase the depth as needed

Reply:
Does this not break the frontend?
Also I'd rather use API URLs rather than primary keys, it's more RESTy - see #272